install.sh

```diff
@@ -42,8 +42,8 @@ esac
 sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
 # Delete 'var' symlink on Cloudready platform
 if [[ $(grep neverware /etc/lsb-release) != "" ]]; then
-  sudo unlink /usr/local/var
-  sudo unlink /usr/local/local
+  if [ -L /usr/local/var ]; then sudo unlink /usr/local/var; fi
+  [ -L /usr/local/local ] && sudo unlink /usr/local/local
```

Comment:
Let's be consistent:
```
[ -L /usr/local/local ] && sudo rm -f /usr/local/local
[ -L /usr/local/var ] && sudo rm -f /usr/local/var
```
It's probably better to use `rm` instead of `unlink` for POSIX portability across systems.

Reply:
Done